ddtrace/tracer/rules_sampler.go

```diff
@@ -97,6 +100,17 @@ func (sr *SamplingRule) match(s *span) bool {
 	} else if sr.exactName != "" && sr.exactName != s.Name {
 		return false
 	}
+	if sr.Resource != nil && !sr.Resource.MatchString(s.Resource) {
+		return false
+	}
+	if sr.Tags != nil {
+		for k, regex := range sr.Tags {
+			v, ok := s.Meta[k]
+			if !ok || !regex.MatchString(v) {
+				return false
+			}
+		}
```

Comment:
It looks like this should match both `Meta` and `Metrics`.

It's not specified in the RFC how numbers should be represented. Maybe talk with the authors about how this should work.